relayer/src/channel.rs

```diff
@@ -1119,7 +1119,7 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Channel<ChainA, ChainB> {
     pub fn build_chan_close_init_and_send(&self) -> Result<IbcEvent, ChannelError> {
         let dst_msgs = self.build_chan_close_init()?;
 
-        let tm = TrackedMsgs::new(dst_msgs, "create channel");
+        let tm = TrackedMsgs::new(dst_msgs, "create channel with ChannelCloseInit");
```

Comment:
I think there's a typo here: the channel is not being created. But we can skip the "create channel" part altogether.
```suggestion
        let tm = TrackedMsgs::new(dst_msgs, "ChannelCloseInit");
```

Reply:
Yes, I just mechanically changed things without noticing that this line looks out of place. Will fix together with the other messages.